ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/actions/execute/ExecuteQueryAction.java

```diff
@@ -80,7 +84,16 @@ public void actionPerformed(AnActionEvent e) {
             return;
         }
 
-        ExecuteQueryPayload executeQueryPayload = new ExecuteQueryPayload(content, editor);
+        Map<String, Object> parameters;
+        try {
+            ParametersService service = ServiceManager.getService(project, ParametersService.class);
+            parameters = service.getParameters();
+        } catch (Exception exception) {
+            registerParametersRetrievalErrorEvent(messageBus, exception, editor);
```

Comment:
It's not "register".
I think that "sendParatemetersRetrievalErrorEvent" would be better name.